flow-server/src/main/java/com/vaadin/flow/server/frontend/BundleUtils.java

```diff
@@ -113,6 +113,18 @@ public static String getChunkId(String className) {
         return StringUtil.getHash(className, StandardCharsets.UTF_8);
     }
 
+    /**
+     * Returns whether the application uses pre-compiled production bundle or a
+     * custom bundle.
+     *
+     * @return <code>true</code> in case of pre-compiled bundle,
+     *         <code>false</code> otherwise
+     */
+    public static boolean isPreCompiledBundle() {
```

Comment:
```suggestion
    public static boolean isPreCompiledProductionBundle() {
```

or is this also for dev bundle?

Reply:
Done